test/client/before-test.js

```diff
@@ -138,6 +147,38 @@
         ok(passed);
     };
 
+    window.createTestIframe = function (attrs, parent) {
+        var iframe = document.createElement('iframe');
+
+        iframe.id = 'test' + Date.now();
+
+        if (typeof attrs === 'string')
```

Comment:
I propose to drop this and always specify the full notation.
It is more readable.

Compare:
```js
window.createTestIframe('');
window.createTestIframe('about:blank');
window.createTestIframe(window.getSameDomainPageUrl(''))
versus

window.createTestIframe({ src:'' });
window.createTestIframe({ src: 'about:blank' });
window.createTestIframe({ src: window.getSameDomainPageUrl('') })

```